api/build.gradle

```diff
@@ -20,7 +20,8 @@ buildscript {
         OKHTTP_VERSION = '2.7.5'
         OPENCENSUS_VERSION = '0.31.1'
         SPRINGFOX_VERSION = '3.0.0'
-        SPRING_BOOT_VERSION = '2.3.12.RELEASE'
+        SPRING_BOOT_VERSION = '2.7.1'
+        SPRING_FRAMEWORK_VERSION = '5.2.22.RELEASE'
```

Comment:
versions below 5.2.20 are vulnerable - without specifying this, gradle would choose 5.2.15